content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -254,6 +259,9 @@ indicates one of the following reasons for the Pod termination:
 `DeletionByPodGC`
 : Pod, that is bound to a no longer existing Node, is due to be deleted by [Pod garbage collection](/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection).
 
+`TerminationByKubelet`
+: Kubelet terminates the Pod, because of the {{<glossary_tooltip term_id="node-pressure-eviction" text="Node Pressure Eviction">}} or the [Graceful Node Shutdown](/docs/concepts/architecture/nodes/#graceful-node-shutdown). Pod is due to be deleted by the controller owning the Pod.
```

Comment:
(nit)
This even is better written as:
```suggestion
: A kubelet requested termination of the Pod, because of either {{<glossary_tooltip term_id="node-pressure-eviction" text="node pressure eviction">}} or [graceful node shutdown](/docs/concepts/architecture/nodes/#graceful-node-shutdown).
```